FluentTerminal.App.Services/Implementation/DefaultValueProvider.cs

```diff
@@ -305,6 +305,7 @@ public TerminalOptions GetDefaultTerminalOptions()
                 FontFamily = "Consolas",
                 FontSize = 13,
                 BackgroundOpacity = 0.8,
+                Padding: 12,
```

Comment:
Not sure if and why it worked for you, but I think this should be an `=` instead of a `:`

```suggestion
                Padding = 12,
```

Reply:
Sorry, my bad. I forgot to push the latest commit.